src/context/sessionContext/index.tsx

```diff
@@ -64,6 +65,8 @@ export interface ISessionProvider {
   session?: Session;
   sessionRequestInProgress?: boolean;
   onError?: (error: Error) => void;
+  restorePreviousSession?: boolean;
+  onSessionRestoreCallback?: (url: string) => void;
```

Comment:
This should be `onSessionRestore` to match `onError` and other events - `onSessionRestore: onSessionRestoreCallback` will let you rename the variable so it doesn't conflict with the onSessionRestore defined above.

Reply:
Fixed in 9812a2bfac650605d92790b7c073fd13ba2f3e7e